libs/pika/synchronization/tests/unit/barrier.cpp

```diff
@@ -1,104 +1,219 @@
-//  Copyright (c) 2007-2012 Hartmut Kaiser
-//  Copyright (c) 2011-2012 Bryce Adelstein-Lelbach
+//  Copyright (c) 2020 Hartmut Kaiser
```

Comment:
Did this change because what actually happened here was that you moved another test in place of this one, not that you changed this file itself directly?